src/main/scala/is/hail/expr/ir/functions/MathFunctions.scala

```diff
@@ -67,6 +68,8 @@ object MathFunctions extends RegistryFunctions {
     else
       r.get
   }
+  
+  def irentropy(s: String): Double = entropy(s)
```

Comment:
does this do something different from `stats.entropy`? I think you can just do `registerWrappedScalaFunction` directly on that.

Reply:
That didn't work because stats.entropy is generic to Iterable[T]:
`java.lang.NoSuchMethodException: is.hail.stats.package$.entropy(java.lang.String)`

Reply:
ok, cool. makes sense.